src/pages/VolunteersJobListingFS.page

```diff
@@ -156,11 +158,17 @@
             <input type="button" value="{!$Label.labelButtonSignUp}"  class="cssJobShiftSignup btn" onclick="OpenSignUpDlg(event.clientX, event.clientY, '{!job.Id}', '', '{!JSENCODE(job.Name)}', ''); return false;"/>
         </apex:outputPanel>
         
-        <apex:outputPanel id="panelExternalJob" rendered="{!job.External_Signup_Url__c != null}" >
+        <apex:outputPanel id="panelExternalJob" rendered="{!job.External_Signup_Url__c != null && NOT(eventAllShiftsInThePast)}" >
         	<apex:outputLink target="_blank" styleClass="externalJobLink" value="{!job.External_Signup_Url__c}" >{!$Label.labelExternalSignupUrl}</apex:outputLink>
         	<br /><br />
         </apex:outputPanel>
-            
+
+        <apex:outputPanel id="panelEventInThePast" rendered="{!eventAllShiftsInThePast}" >
+        	<apex:outputText value="{!$Label.labelEventInThePast}" />
+        	<apex:outputText rendered="{!job.First_Shift__c != null}" value=" First Shift: " />
+        	<apex:outputText rendered="{!job.First_Shift__c != null}" value="{0,date,MM/dd/yyyy}"><apex:param value="{!job.First_Shift__c}" /></apex:outputText>
```

Comment:
You need to do the correct date formatting based on the language string that is specified as a URL parameter.  I have a javascript helper on this page to do this.  look for this call in the code to see how to use it:

var strStartDate = MyDateFormat(dtStart, '{!JSENCODE(strDateFormat)}');